esphome/components/modbus_controller/modbus_controller.h

```diff
@@ -322,9 +321,9 @@ class ModbusCommandItem {
       on_data_func;
   std::vector<uint8_t> payload = {};
   bool send();
-  // wrong commands (esp. custom commands) can block the send queue
-  // limit the number of repeats
-  uint8_t send_countdown{MAX_SEND_REPEATS};
+  /// Check if the command should be retried based on the max_send_count parameter
+  bool should_retry(uint8_t max_send_count) { return this->send_count_ < max_send_count; };
```

Comment:
Change the test so the first one always gets sent.
```suggestion
  bool should_retry(uint8_t max_retries) { return this->send_count_ <= max_retries; };
```

Reply:
In this case we need `max_cmd_retries_` to be 5, but it would represent 4 retries. Am I right? So I can update the documentation.

Reply:
Correct. One initial send, 4 retries. On the basis that there will always be the initial send.